sensor_msgs/Image image
---
bool success
context_action_framework/Detection[] detections
sensor_msgs/Image labelled_image
sensor_msgs/Image cropped_image